### ISSUE TITLE
openssl version 1.1.1 and Qt 5.12.4/5

### ISSUE THREAD

**Issue by BlockMechanic** [bitcoin/bitcoin#17123]
I noticed while trying to deploy for all platforms on Qt 5.12.4 that as of that version only openssl version 1.1.1+ is supported. This prompted me to check what version is used currently in this project (1.0.1k) and to also check the EOL. 

I found that 1.0.1 version reached EOL in 2016. 1.0.2 reaches EOL in december 2019 so I wanted to know if there is a reason the default is not  1.1.1 ? This also probably supports our efforts to encourage a bump from QT 5.9 to 5.12.4/5. I think a comprehensive look across all platforms should be considered.

**Comment by rashid8080** [comment]
Can't exec "aclocal": No such file or directory at /data/data/com.termux/files/usr/share/autoconf/Autom4te/FileUtils.pm line 326.

**Comment by rashid8080** [comment]
I need your help

**Comment by laanwj** [comment]
Note that all the code that uses OpenSSL is deprecated (part of BIP70 deprecation). I'd hope we can get rid of the dependency before having to bump it.

**Comment by BlockMechanic** [comment]
Ah, that would be great, the less dependencies the better. Since it's already on its way out, i'll just build 
 QT without openssl and drop bip70 support completely, especially in the android builds i am working on.

**Comment by laanwj** [comment]
I think this is no longer relevant after #17265.
